cmd/addoperator.go

```diff
@@ -330,6 +333,9 @@ func (p *AddOperatorParams) Run(_ ActionCtx) (store.Status, error) {
 		if err == nil {
 			ocNew, err := jwt.DecodeOperatorClaims(p.token)
 			if err != nil {
+				if err.Error() == JWTV2DecodeError {
+					r.AddError("%sfailed to decode: %v", JWTUpgradeBannerJWT(), err)
```

Comment:
possibly - remove the new line from the format, since you are just adding a line, and add the \n here.